### ISSUE TITLE
Disable Split GPG while screen locker is active

### ISSUE THREAD

**Issue by andrewdavidwong** [QubesOS/qubes-issues#2443]
[On 2016-11-18 09:39, Michael Carbone wrote:](https://groups.google.com/d/msgid/qubes-users/e391c797-5076-2955-77a1-597ebf302b9e%40qubes-os.org)
> Marek Marczykowski-Górecki:
>> On Fri, Nov 18, 2016 at 02:49:00PM +0000, Michael Carbone wrote:
>>> From a security perspective without timestamps in the access logs
>>> (https://github.com/QubesOS/qubes-issues/issues/1835) a malicious
>>> pre-approved email client could just decrypt emails in mass when the
>>> user is AFK to avoid notifying the user, so I see little security benefit.
>>
>> That's true indeed. I wonder if blocking split-gpg while screenlocker is
>> engaged would make sense? Currently similar purpose have confirmation
>> with a 5min timeout.
> 
> I think that's an excellent idea.
>

**Comment by o-** [comment]
Hi, I just stumbled upon this issue. I think it is certainly a good idea. I just wanted to note, that there are usecases for using split-gpg in the background (for example indexing a large mailbox with notmuch). If possible it should be made configurable somehow.

**Comment by ben-grande** [comment]
I don't think it is a good idea, at least not by default. I have to constantly interact with split-gpg2 and prompts are an annoyance and can be a burden and dismissed if I am not attentive all the time. It is better to disable them... I don't oppose having an option disabled by default for this matter. It needs a Qrexec service targeting the GUI domain to query if screen locker is active and must be independent of the screen locker tool.
